contracts/cheddar_5x5_tic_tac_toe/src/game.rs

```diff
@@ -37,27 +36,32 @@ pub struct Game {
 impl Game {
     /// set players in given order. First player (`player_1`)
     /// will have first move
-    /// It generates randomly in `Contract.start_game` because 
+    /// It generates randomly in `Contract.start_game` because
     /// first move gives more chances to win
     pub fn create_game(
+        game_id: GameId,
         player_1: AccountId,
         player_2: AccountId,
-        reward: GameDeposit
+        reward: GameDeposit,
     ) -> Game {
-        assert_ne!(player_1, player_2, "Player 1 and Player 2 have the same AccountId: @{}", &player_1);
+        assert_ne!(
+            player_1, player_2,
+            "Player 1 and Player 2 have the same AccountId: @{}",
+            &player_1
+        );
         let (player_1, player_2) = Game::create_players(player_1, player_2);
-        let board = Board::new(&player_1, &player_2);
-        let mut game = Game { 
-            game_state: GameState::NotStarted, 
-            players:Vec::with_capacity(PLAYERS_NUM),
+        let board = Board::new(game_id, &player_1, &player_2);
+        let mut game = Game {
+            game_state: GameState::NotStarted,
+            players: Vec::with_capacity(PLAYERS_NUM),
```

Comment:
let's use pair here instead of Vector. So change type from `Vec<AccountId` to `(AccountId, AccountId)`. To access the pair you can use `players.1` (the second player).